internal-common/src/main/java/net/kyori/moonshine/internal/ReflectiveUtils.java

```diff
@@ -55,11 +57,9 @@ private static String formatMethodParameters(final Method method) {
       return "";
```

Comment:
Early return is redundant imo. Performance isn't critical for this method and the stream solution *should* (please actually verify this) work fine for empty arrays.

Reply:
I'd prefer to keep the early return to show the behaviour clearly and also avoid all the allocations of a string joiner, streams, etc.